sn_networking/src/record_store.rs

```diff
@@ -975,7 +1005,7 @@ mod tests {
             if iteration == 50 {
                 assert_eq!(0, empty_earned_nodes);
                 assert!((max_store_cost / min_store_cost) < 40);
-                assert!((max_earned / min_earned) < 400);
+                assert!((max_earned / min_earned) < 600);
```

Comment:
This change is nothing related to this PR,
just seeing sparadically failure of unit test due to too large earned variant.